extensions/amp-autocomplete/validator-amp-autocomplete.protoascii

```diff
@@ -106,3 +106,14 @@ tags: {  # amp-autocomplete JSON
     }
   }
 }
+tags: {  # amp-autocomplete disabled
+  html_format: AMP
+  tag_name: "DIV"
+  spec_name: "amp-autocomplete disabled"
```

Comment:
We tend to write these as `spec_name: "amp-autocomplete > div[disabled]"`, also in the comment at line 109.